sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/producer/KafkaMessageProducer.java

```diff
@@ -45,4 +45,9 @@ public Future<RecordMetadata> send(K key, V value, Headers headers) {
   public void close() {
     producer.close();
   }
+
+  @Override
+  public void flush() {
+    producer.flush();
+  }
```

Comment:
Can you add a simple Mockito test to assert that flush is delegated to the producer?